sdk/test-utils/perfstress/src/tests.ts

```diff
@@ -27,6 +35,8 @@ export interface PerfStressTestConstructor<TOptions extends {} = {}> {
  * (initializations are as many as the "parallel" command line parameter specifies).
  */
 export abstract class PerfStressTest<TOptions = {}> {
+  public static testProxyHttpClientV1: TestProxyHttpClientV1 | DefaultHttpClient;
+  public static testProxyHttpClient: TestProxyHttpClient;
```

Comment:
Can we not do statics? I would greatly prefer you expose your getter methods (and do some internal caching with private properties)

Reply:
Can do.
Just curious.. why not statics?

Reply:
Updated